collector.go

```diff
@@ -0,0 +1,94 @@
+package redigosrv
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+//RedigoCollector struct to access metrics
+type RedigoCollector struct {
+	publishTrafficSize prometheus.Counter
+	methodCalls        *prometheus.CounterVec
+	subscribeAmount    prometheus.Gauge
```

Comment:
I think this would be a more suitable name for this metric.

```suggestion
	subscriptionsActive    prometheus.Gauge
```